### ISSUE TITLE
Add classes for primary-color, secondary-color, accent-color, primary-color-text, secondary-color-text and accent-color-text 

### ISSUE THREAD

**Issue by rascoop** [Dogfalo/materialize#4273]
### Description
Make it easier for the end users to apply the correct color for the color scheme without knowing what these are.

**Comment by fega** [comment]
I always consider that the color features add technical debt to the project, I agree with this enhancement. or at least make the recommendation in the official docs

Also, I want to see the color feature ridden to materialize, but I know it would be a major breaking change

**Comment by BonnieDoug** [comment]
100% yes to this.

**Comment by tomscholz** [comment]
I think this would break many sass installation and would be a big change. @fega @acburst @Dogfalo @DanielRuf What do you think?

**Comment by DanielRuf** [comment]
At the moment there are no accent color variables. So this would introduce some bigger changes in the sass files and the documentation.

I'm with the opinion of @tomscholz.

Or am I wrong?

**Comment by fega** [comment]
@DanielRuf yeah, I don't think so, we can create some 'Placeholder variables' with optional use, and describe them in the docs.
